### ISSUE TITLE
Latest release break

### ISSUE THREAD

**Issue by raianul** [xmlsec/python-xmlsec#318]
```FROM python:3.10.8-slim

RUN apt update -y && \
    apt install -y gcc libxml2-dev libxmlsec1-dev libxmlsec1-openssl pkg-config libpq-dev && \
    rm -rf /var/lib/apt/lists/* && \
    python3 -m pip install --upgrade pip setuptools wheel lxml xmlsec```


The latest release breaking the install. 

```laration-after-statement -Werror=implicit-function-declaration -Os
5.006       /tmp/pip-install-w8qtbzg3/xmlsec_9b3ec92e2bd84a5d86e839c6e575639e/src/keys.c: In function ‘PyXmlSec_KeyFromEngine’:
5.006       /tmp/pip-install-w8qtbzg3/xmlsec_9b3ec92e2bd84a5d86e839c6e575639e/src/keys.c:218:65: error: ‘xmlSecKeyDataFormatEngine’ undeclared (first use in this function); did you mean ‘xmlSecKeyDataFormat’?
5.006         218 |         key->handle = xmlSecCryptoAppKeyLoad(engine_and_key_id, xmlSecKeyDataFormatEngine, NULL, xmlSecCryptoAppGetDefaultPwdCallback(), (void*)engine_and_key_id);
5.006             |                                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~
5.006             |                                                                 xmlSecKeyDataFormat
5.006       /tmp/pip-install-w8qtbzg3/xmlsec_9b3ec92e2bd84a5d86e839c6e575639e/src/keys.c:218:65: note: each undeclared identifier is reported only once for each function it appears in
5.006       error: command '/usr/bin/gcc' failed with exit code 1
5.006       [end of output]```

**Comment by mxamin** [comment]
Duplicate of https://github.com/xmlsec/python-xmlsec/issues/314.
